packages/wrangler/src/__tests__/init.test.ts

```diff
@@ -940,7 +948,7 @@ describe("init", () => {
           "debug": "",
           "err": "",
           "out": "✨ Created wrangler.toml
-        ✨ Created tsconfig.json, installed @cloudflare/workers-types into devDependencies",
```

Comment:
Why do some of these no longer install the workers-types into devDependencies? Or at least no longer logs that it is doing so?

Reply:
Great call-out! I have no idea 😨 , will look into it.

Reply:
Turns out I missed a few use-cases @petebacondarwin - I added a catch-all run of installPackages (for when we don't write package.json scripts, for example